assets/src/edit-story/components/canvas/karma/cloneSelection.karma.js

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { useStory } from '../../../app/story';
+
+describe('Clone element integration', () => {
+  let fixture;
+  let bg, img1, img2;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+    jasmine.addMatchers(customMatchers);
+
+    bg = await getElementByIndex(1);
+    const bgRect = (
+      await getCanvasElementWrapperById(bg.id)
+    ).getBoundingClientRect();
+
+    // Add first image to page
+    await fixture.events.click(getMediaElement(/blue-marble/));
+    img1 = await getElementByIndex(2);
+    // Drag it to (50,50)
+    const rect1 = (
+      await getCanvasElementWrapperById(img1.id)
+    ).getBoundingClientRect();
+    await fixture.events.mouse.seq(({ move, down, up }) => [
+      move(rect1.left + 10, rect1.top + 10),
+      down(),
+      move(bgRect.left + 50, bgRect.top + 50, { steps: 2 }),
+      up(),
+    ]);
```

Comment:
The need for dragging here is to be able to correctly select the items later by clicking them in the top left corner - they need to not overlap there (and as they're currently placed randomly on insertion, it failed sporadically when they happened to overlap).